open_ce/inputs.py

```diff
@@ -363,6 +373,26 @@ def _create_env_config_paths(args):
                 log.info("Unable to find '%s' locally. Attempting to use '%s'.", config_file, new_url)
                 args.env_config_file[index] = new_url
 
+def _check_ppc_arch(args):
+    '''
+    This will check if ppc_arch is p10 and set the corresponding
+    needed environment variables for GCC_10_HOME and GCC_11_HOME
+    '''
+    if "ppc_arch" in vars(args).keys() and args.ppc_arch:
+        if args.ppc_arch == "p10":
+            if "GCC_10_HOME" not in os.environ:
+                os.environ["GCC_10_HOME"] = utils.DEFAULT_GCC_10_HOME_DIR
+            if os.path.exists(os.environ["GCC_10_HOME"]):
+                PATH = os.environ["PATH"]
+                os.environ["PATH"] = "{0}:{1}".format(os.path.join(os.environ["GCC_10_HOME"], "bin"), PATH)
+                print("Path variable set to : ", os.environ["PATH"])
+            else:
+                raise OpenCEError(Error.GCC10_11_COMPILER_NOT_FOUND)
+            if "GCC_11_HOME" not in os.environ and os.path.exists(utils.DEFAULT_GCC_11_HOME_DIR):
+                os.environ["GCC_11_HOME"] = utils.DEFAULT_GCC_11_HOME_DIR
+            else:
+                raise OpenCEError(Error.GCC10_11_COMPILER_NOT_FOUND)
```

Comment:
I think this needs to be split into two steps like for GCC_10_HOME, as presently the else block gets called when GCC_11_HOME is defined.

Reply:
Yes, thanks!